test/src/e2e_vm_tests/test_programs/should_pass/language/u64_ops/Forc.toml

```
@@ -0,0 +1,9 @@
+[project]
+authors = ["Fuel Labs <contact@fuel.sh>"]
+entry = "main.sw"
+license = "Apache-2.0"
+name = "u64_ops"
+
+[dependencies]
+core = { path = "../../../../../../../sway-lib-core" }
+std = { path = "../../../../../../../sway-lib-std" }
```

`core` is redundant.
```suggestion
std = { path = "../../../../../../../sway-lib-std" }
```

---

@adlerjohn I I remove the core import here I get this error:
```
thread 'main' panicked at 'no entry found for key', forc-pkg/src/pkg.rs:539:45
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

---

That sounds like a bug >.< !

---

hmm.  @mitchmindtree what're your thoughts here?

---

I feel like it might be related to https://github.com/FuelLabs/sway/issues/1172?

---

@nfurfaro nice find!

I've opened https://github.com/FuelLabs/sway/pull/1314 - see that PR for details on what's going on here. I've tested it locally with your branch (after removing `core`) and can no longer recreate the issue with that patch.

In the meantime you should be able to delete your local copy of the `Forc.lock` file and then `forc build` will reconstruct, a correct `Forc.lock` file, side-stepping this issue.